layouts/blog/single.html

```diff
@@ -1,37 +1,69 @@
+<article class="post" itemscope itemtype="http://schema.org/BlogPosting">
+  <div class="header-content">
+    <h1>{{ .Params.title }}</h1>
+    <p>{{ .Params.description }}</p>
+  </div>
+  <p class="post-author">
+    {{ (.Params.publishdate).Format "Jan 2, 2006" }} <span> | </span> By {{
+    replace (replace .Params.attribution "(" "- ") ")" "" }}
+  </p>
+  {{ if .Page.Params.target_release }} {{ $page_parts := split
+  .Page.Params.target_release "." }} {{ $page_version := int (index $page_parts
+  0) }} {{ $page_revision := int (index $page_parts 1) }} {{
+  $page_normalized_version := add (add (mul $page_version 1000000) (mul
+  $page_revision 1000)) 0 }} {{ $site_parts := split .Site.Data.args.version "."
+  }} {{ $site_version := int (index $site_parts 0) }} {{ $site_revision := int
+  (index $site_parts 1) }} {{ $site_normalized_version := add (add (mul
+  $site_version 1000000) (mul $site_revision 1000)) 0 }} {{ if gt
+  $site_normalized_version $page_normalized_version }}
+  <div>
+    <aside class="callout warning">
+      <div class="type">
+        {{- partial "large_icon.html" "callout-warning" -}}
       </div>
-      <p class="post-author">{{ (.Params.publishdate).Format "Jan 2, 2006" }} <span> | </span> By {{ replace (replace .Params.attribution "(" "- ") ")" "" }}</p>
-      {{ if .Page.Params.target_release }}
-          {{ $page_parts := split .Page.Params.target_release "." }}
-          {{ $page_version := int (index $page_parts 0) }}
-          {{ $page_revision := int (index $page_parts 1) }}
-          {{ $page_normalized_version := add (add (mul $page_version 1000000) (mul $page_revision 1000)) 0 }}
-
-          {{ $site_parts := split .Site.Data.args.version "." }}
-          {{ $site_version := int (index $site_parts 0) }}
-          {{ $site_revision := int (index $site_parts 1) }}
-          {{ $site_normalized_version := add (add (mul $site_version 1000000) (mul $site_revision 1000)) 0 }}
-
-          {{ if gt $site_normalized_version $page_normalized_version }}
-              <div>
-                  <aside class="callout warning">
-                      <div class="type">
-                          {{- partial "large_icon.html" "callout-warning" -}}
-                      </div>
-                      <div class="content">
-                          {{ printf (i18n "target_release") .Page.Params.target_release }}
-                      </div>
-                  </aside>
-              </div>
-          {{ end }}
-      {{ end }}
-      <div>
-        {{ .Content }}
+      <div class="content">
+        {{ printf (i18n "target_release") .Page.Params.target_release }}
       </div>
-      {{ partial "posts_navigation.html" . }}
-    </article>
-{{ end }}
\ No newline at end of file
+    </aside>
+  </div>
+  {{ end }} {{ end }}
+  <div>{{ .Content }}</div>
+  <div class="social-share">
+    <p class="social-share__title"><h4>SHARE</h4></p>
+    <div class="social-share__wrapper">
```

Comment:
You'll need to change the SASS as the class `.social-share` is set to `display: none !important` which hides this entire div. If you run `make serve` locally, it will run hugo which will display the site so you can try this out locally.

If you click "Details" next to the "test" `netlify/preliminary-istio/deploy-preview — Deploy Preview ready! ` you'll be taken to the live prelim view of this PR.

Reply:
I don't see these other classes defined, so if they need to be they should be added otherwise remove them from here as they aren't needed.